bin/dataproc

```diff
@@ -25,16 +26,20 @@ function bootstrap() {
     tee bootstrap/install-python-requirements.sh >/dev/null <<EOF
 #!/bin/bash
 apt update && apt install --yes python-dev libmsgpackc2 libnss3
-gsutil cp ${bucket}/bootstrap/requirements.txt .
+gsutil cp ${bucket}/requirements.txt .
 pip install -r requirements.txt
 EOF
-    tee bootstrap/processor.py >/dev/null <<EOF
-from prio_processor.spark import commands
+    tee bootstrap/processor-spark.py >/dev/null <<EOF
+from ${MODULE}.spark import commands
+commands.entry_point()
+EOF
+    tee bootstrap/processor-origin.py >/dev/null <<EOF
+from ${MODULE}.origin import commands
```

Comment:
There's a separate runner for each module now, since they play entire different roles in the processing pipeline.